src/redux/selectors.js

```diff
@@ -11,7 +11,11 @@ const getBackendStatus = (...params) => (state) => (
 const getCellSets = (...params) => (state) => (
   cellSetsSelectors.getCellSets(...params)(state.cellSets));
 
+const getCellSetsHierarchy = (cellSets, category) => (
+  cellSetsSelectors.getCellSetsHierarchy(cellSets, category));
```

Comment:
This selector should work like any other selector, take the `category` param, and return a function that takes the state